### DISCUSSION TITLE
Moved storage location and lost all settings, api keys, users etc.

### DISCUSSION THREAD

**Discussion by rajbindu** [datalust/seq-tickets#1818] [Help]
Hi,

We moved the storage location of seq using the instructions in the following link, but now all settings are lost. Please help!

https://docs.datalust.co/docs/storage-paths-and-urls

Thanks
Raj

**Comment by nblumhardt** [comment]
Hi Raj, thanks for getting in touch, and sorry to hear about the difficulties.

Can you please send more info on your Seq setup, ideally to support@datalust.co:

 * Seq version
 * Operating system
 * Old folder path
 * New folder path (ideally screenshot of directory contents)
 * Output from `seq service list` at an admin command prompt
 * Company name for your Seq subscription

Thanks!

Best regards,
Nick
